bridge-token-factory/readme.md

```diff
@@ -0,0 +1,28 @@
+# Bridge-Token-Factory
+---
+## Fee Integration for deposit and withdrawal of tokens
+**NOTE**:
+1. To set fee-percentage and fee-bounds caller (predecessor) must have `FeeSetter` role.
+2. Fee-percentage has **6-decimal** precision for eg. to set deposit fee percentage as 10%, value for function param is 0.1 * 10^6 ie. 10^5.
+
+* ### For deposit: 
+  
+  * `set_deposit_fee_percentage`: method to set deposit fee-percentage for eth -> near and eth -> aurora.
+  * `set_deposit_fee_bound`: method to set upper and lower_bounds for deposit fee.
+  * `get_deposit_token_fee_percentage`: returns deposit token fee percentage for specific erc-20 token address passed.
+  * `get_deposit_token_fee_bound`: returns fee-bounds for deposits of specific erc-20 token address.
+
```

Comment:
Now we also have function `set_deposit_fee_per_silo` 

The README should be updated according current implementation.

Reply:
849520f130778a2f9f143d3b37d33d8299f2eda0